seahorn/lib/proof_allocators.c

```diff
@@ -40,17 +40,49 @@ static void s_free_allocator(struct aws_allocator *allocator, void *ptr) {
   free(ptr);
 }
 
+static void *s_realloc_allocator(struct aws_allocator *allocator, void *ptr, size_t oldsize, size_t newsize) {
```

Comment:
can we use this realloc function

https://sourcegraph.com/github.com/seahorn/verify-c-common/-/blob/seahorn/lib/sea_allocators.c?L72&subtree=true

Reply:
I do not get your point. The `s_realloc_allocator` has a different function type from `sea_realloc`.  Do you mean this:
We change
```c
void *new_mem = s_malloc_allocator(allocator, newsize);
```
into
```c
void *new_mem = sea_realloc(ptr, newsize);
```